.github/workflows/build.yml

```diff
@@ -35,3 +35,10 @@ jobs:
         with:
           name: 'backstop_test_results'
           path: '${{ github.workspace }}/backstop/results/'
+
+      - name: Host test results on gh pages
+        uses: peaceiris/actions-gh-pages@v3.7.3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ${{ github.workspace }}/backstop/results
```

Comment:
@veekeys I see you changed the folder to `results/` rather than `results/html_report/`. Any reason?

Reply:
Oh, I see the images are located outside that folder... That's not nice 😕 
Maybe we need some redirect